vcluster/deploy/topologies/high-availability.mdx

```diff
@@ -27,6 +27,10 @@ controlPlane:
     highAvailability:
       replicas: 3
 ```
+:::info
```

Comment:
Instead of giving all the other options, maybe we simplify it to:
```
High Availability mode is not compatible with the embedded SQLite as the backing store, which is the default backing store. You must explicitly set an alternative [backing store](../../configure/vcluster-yaml/control-plane/components/backing-store).
```